doc/lib/platform.txt

```diff
@@ -10,13 +10,50 @@ All system calls (including networking) used in this SDK's libraries go through
   @copybrief platform_threads
 - @ref platform_network <br>
   @copybrief platform_network
+- @ref platform_metrics <br>
+  <i>Only used by [Device Defender](@ref defender); does not need to be ported if not using Defender.</i> <br>
+  @copybrief platform_metrics <br>
+- @ref platform_atomic <br>
+  @copybrief platform_atomic
 
-All of the functions in each component will need to be implemented to port the libraries to a new platform. Currently, implementations exist for the following platforms:
+Currently, implementations exist for the following platforms:
 Component                   | Supported platforms
 ---------                   | -------------------
-@ref platform_clock         | [POSIX](https://en.wikipedia.org/wiki/POSIX)
-@ref platform_threads       | [POSIX](https://en.wikipedia.org/wiki/POSIX)
-@ref platform_network       | [Berkeley sockets](https://en.wikipedia.org/wiki/Berkeley_sockets) + [OpenSSL](https://en.wikipedia.org/wiki/OpenSSL) + [POSIX](https://en.wikipedia.org/wiki/POSIX) <br> [Amazon FreeRTOS Secure Sockets](https://docs.aws.amazon.com/freertos/latest/userguide/secure-sockets.html)
+@ref platform_clock         | [FreeRTOS Software Timers](https://www.freertos.org/RTOS-software-timer.html)
```

Comment:
Synchronized this with CSDK and update for Amazon FreeRTOS's current implementations.